inc/template.php

```diff
@@ -985,6 +985,9 @@ function tpl_pagetitle($id = null, $ret = false)
         $first_heading = p_get_first_heading($id);
         if ($first_heading) $name = $first_heading;
     }
+    if (auth_quickaclcheck($id) < AUTH_READ) {
+        $name = str_replace('.', '', $lang['accessdenied']);
```

Comment:
So if the message happens to consist of two sentences then you remove the both periods? And what about other punctuation? What if someone ends the message with an exclamation mark? And who knows what conventions exist in other languages?

A localizable string should only serve one semantic purpose.

Reply:
Now it uses a ``preg_replace()`` that removes dot from the end of the string on LTR and RTL languages.
If any language uses any other character like exclamation point, there's no problem. This ``preg_replace()`` is just to format the string, there's no real need to remove it from it. The intention of the text on the ``title`` HTML tag is to be short, so I do not believe anyone would create a string with two or more sentences on it, because the original translation isn't.

Reply:
I still strongly disagree with the concept of modifying localized text. That is something the localizers do. Doing so in code makes assumptions that may not be true.

I also don't see any reason to not add a dedicated localised string for this.